lib/instrumentation/http-shared.js

```diff
@@ -169,13 +169,8 @@ exports.traceOutgoingRequest = function (agent, moduleName, method) {
       var req = orig.apply(this, newArgs)
       if (!span) return req
 
-      if (getSafeHost(req) === agent._conf.serverHost) {
-        agent.logger.debug('ignore %s request to intake API %o', moduleName, { id: id })
```

Comment:
NOTE: No longer need this guard.

Reply:
Agreed -- just to say it out loud removing this guard clause does mean if a user has an application that

1. Was instrumented by the agent
2. Was making their own calls to APM Server

that they'll suddenly see spans for those requests when they wouldn't previously. 

This case is pretty far fetched though and "more span" isn't exactly a bad thing.